airbyte-workers/src/main/java/io/airbyte/workers/general/DefaultReplicationWorker.java

```diff
@@ -128,6 +128,12 @@ public final ReplicationOutput run(final StandardSyncInput syncInput, final Path
     destinationConfig.setCatalog(mapper.mapCatalog(destinationConfig.getCatalog()));
 
     final long startTime = System.currentTimeMillis();
+    final long replicationStartTime = startTime;
+    long replicationEndTime = -1;
+    long sourceReadStartTime = -1;
+    long destinationWriteStartTime = -1;
```

Comment:
It seems that these longs needed to be initialized or else they couldn't be used below in `withSourceReadStartTime()` - I'll be there's an `Optional<>` Java thing I don't know about that's better than all the ternaries...